xbmc/video/tags/VideoTagLoaderFFmpeg.cpp

```diff
@@ -244,5 +244,5 @@ CInfoScanner::INFO_TYPE CVideoTagLoaderFFmpeg::LoadMP4(CVideoInfoTag& tag,
       tag.m_coverArt.emplace_back(EmbeddedArtInfo(size, "image/png", type));
   }
 
-  return CInfoScanner::FULL_NFO;
+  return CInfoScanner::TITLE_NFO;
```

Comment:
this will need some more logic, if there are more tags the assumption must be that user wants to use the tag info only.

Reply:
Indeed the trade-of in between `TITLE_NFO` and `FULL_NFO` is pretty difficult to find. For movies: `title` + `year` is enough, for  music video: `title` + `album_artist` is enough, for TV-show `title` + `show` + `episode_id` is enough. I cannot decide by myself, could you describe your definition of `FULL_NFO` ?

I can put my understand of `FULL_NFO` here for clarity: if either `kodi-metadata` or `kodi-override-metadata` is found, then we can return `FULL_NFO`, otherwise info may be incomplete then return `TITLE_NFO`

For reference:

* https://wiki.multimedia.cx/index.php/FFmpeg_Metadata

Reply:
it is indeed quite tricky. since mkv tags are very badly spec'd, hardly anything is standardized. so the current code is pretty much placeholder, with the kodi specific fields being my hack (you can embed standard kodi nfo files in there) to make it usable at this point.

mp4 however, is more spec'd so i think this block needs to check if we find anything in addition. i don't recall if content type is passed into the loader or not, but that possibly needs addition.

Reply:
@notspiff So if either `description` or `synopsis` is found, assume that user wants to use tag only. Let me know what you think about latest update. Thanks